ha/virsh/tests/resource_lvm-activate_test.sh

```diff
@@ -29,22 +29,22 @@ oneTimeSetUp() {
 }
 
 configure_cluster_properties() {
-  run_command_in_node "${IP_VM01}" "sudo crm configure property stonith-enabled=off"
+  run_command_in_node "${IP_VM01}" "sudo pcs property set stonith-enabled=false"
 }
 
 configure_lvm_activate_resource() {
   NAME="${1}"
   VOLGROUP="${2}"
-  run_command_in_node "${IP_VM01}" "sudo crm configure primitive ${NAME} ocf:heartbeat:LVM-activate \
+  run_command_in_node "${IP_VM01}" "sudo pcs resource create ${NAME} ocf:heartbeat:LVM-activate \
 	  vgname=${VOLGROUP} vg_access_mode=system_id"
 }
 
 test_lvm_activate_is_started() {
   configure_cluster_properties
   configure_lvm_activate_resource "${RESOURCE_NAME}" "${VG}"
 
-  run_command_in_node "${IP_VM01}" "sudo crm cluster wait_for_startup"
-  cluster_status=$(run_command_in_node "${IP_VM01}" "sudo crm status")
+  sleep 10
```

Comment:
`pcs resource create` has a `--wait` option, could that be used in `configure_lvm_activate_resource` when it calls that command? Instead of a `sleep` here?